src/lib/storage/table_key_constraint.cpp

```diff
@@ -2,13 +2,20 @@
 
 namespace opossum {
 
-TableKeyConstraint::TableKeyConstraint(std::unordered_set<ColumnID> init_columns, KeyConstraintType init_key_type)
-    : AbstractTableConstraint(std::move(init_columns)), _key_type(init_key_type) {}
+TableKeyConstraint::TableKeyConstraint(std::unordered_set<ColumnID> init_columns, KeyConstraintType init_key_type,
+                                       std::optional<CommitID> latest_validated_commit /* = std::nullopt*/)
+    : AbstractTableConstraint(std::move(init_columns)),
+      _key_type(init_key_type),
```

Comment:
Not yours, but we actually only use the `init_` prefix in case there is a naming conflict as the member is public. Since key type is private and thus prefixed with `_`, the is no reason to use the `init_` prefix here.

Otherwise it's also not consistent with `latest_validated_commit`.